tests/unit/completion/test_completiondelegate.py

```diff
@@ -0,0 +1,108 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2018 Ryan Roden-Corrent (rcorre) <ryan@rcorre.net>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for the CompletionView Object."""
+
+from unittest import mock
+
+import pytest
+
+from PyQt5.QtGui import QPainter
+
+from qutebrowser.completion import completiondelegate
+
+
+@pytest.fixture
+def painter():
+    """Create the CompletionView used for testing."""
+    return mock.Mock(spec=QPainter)
+
+
+def _qt_mock(klass, mocker):
+    m = mocker.patch(
+        'qutebrowser.completion.completiondelegate.{}'.format(klass),
+        autospec=True)
+    return m
```

Comment:
nitpick: Could probably just return it directly instead of assigning it to a name.